File: extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/xml/persistence/MyEntity.java
package io.quarkus.hibernate.orm.xml.persistence;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

@Entity
public class MyEntity {

    @Id
    public long id;

    public String name;

    public MyEntity() {
    }

    public MyEntity(String name) {
        this.name = name;
    }

}

File: extensions/panache/hibernate-orm-panache-kotlin/deployment/src/test/kotlin/io/quarkus/hibernate/orm/panache/kotlin/deployment/test/MyEntity.kt
package io.quarkus.hibernate.orm.panache.kotlin.deployment.test

import io.quarkus.hibernate.orm.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity
import jakarta.persistence.Entity

@Entity
class MyEntity : PanacheEntity() {
    companion object: PanacheCompanion<MyEntity> {
    }

    lateinit var name: String
}
